app/src/lib/get-os.ts

```diff
@@ -16,3 +16,16 @@ export function getOS() {
     return `${OS.type()} ${OS.release()}`
   }
 }
+
+/** See the OS we're currently running on is at least Mojave. */
+export function isMojaveOrLater() {
+  if (__DARWIN__) {
+    const parser = new UAParser()
+    const os = parser.getOS()
+    // Check that it is Mojave or later
+    if (os.version && os.version.split('.')[1] > '13') {
```

Comment:
I was thinking that too but how should it behave in the case above? I thought it would return false anyway as it is unable to verify Mojave :/

Reply:
>  I thought it would return false anyway as it is unable to verify Mojave :/

Agreed, just thinking out loud here how much we should rely on JS truthiness here.

If we wanted to be explicit here and not trust `os.version` so much, we could do something like this:

```ts
if (os.version == null) { 
  return false
}

const parts = os.version.split('.')
if (parts.length !== 2) {
  // unknown version format, giving up
  return false
}

return parts[1] >  '13'
```

Reply:
Sounds good to me. Shall make the changes shortly